src/main/java/data/exercises/ExerciseList.java

```diff
@@ -25,6 +25,7 @@ public void addExerciseToList(String exerciseName) {
      * Prints the list of exercises starting with their index.
      */
     public void printExerciseList() {
+        assert getNumberOfExercises() == 13;
```

Comment:
In the situation of real app implementation, there will be more exercises in the file so this assertion might constantly change (understandable that assertion can be disabled too). Some suggestions perhaps:
1. Use constant variable instead of magical numbers.
2. Make use of .size() instead?

Reply:
yea, in the real implementation I will update it accordingly, the 13 is just for now, I didn't think of using size(), thanks for the suggestion. I was thinking since the getNumberOfExercises() returns numberOfExercises, it is a bit pointless to do assert getNumberOfExercises() == numberOfExercises; that's why I used 13, I will update it to use .size()! Thank you!